### PR TITLE
Add fix to1 only (ready, other one can be deleted)

### PR DESCRIPTION
**Pull request by EyalMecilati**

### CODE REVIEW

File: src/comps/CompanyComparison/ComparisonUtils.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const getGraphItemByType = (
         fieldsTexts[fieldName].title,
         userProductAndCurrency
       ),
-      format: (x) => x ? x.toString() : ''
+      format: (x) => formatNumberWithCommas(Number(x)) ? formatNumberWithCommas(Number(x))!.toString() : ''
```
Comment (alissaVrk) [inline src/comps/CompanyComparison/ComparisonUtils.ts:44]: it is strange to call the function twice..
why don't you just check if `x` exists?

File: src/comps/companyPage/columnsConfigAndUtils.ts [hunk 2]
```diff
@@ -102,7 +102,16 @@ export const columnsForLom = (product: ProductType) => {
 };
 
 export const formatNumberWithCommas = (num: number) => {
-  return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+  let formattedWithCommas = null
+  if(num){
+    let numFixed:string | number = num;
```
Comment (alissaVrk) [inline src/comps/companyPage/columnsConfigAndUtils.ts:107]: please create another var for the string

File: src/comps/grid/useGridColumns.tsx [hunk 3]
```diff
@@ -103,14 +104,24 @@ export function useGridColumns(colsCategs: Array<ColumnCategory>, companiesData:
...
+              );
             }
-            if(fieldDef.dataType === FieldDataTypes.Quorter){
-              return getFormatQuorter(value)
+            if (fieldDef.dataType === FieldDataTypes.Quorter) {
+              return value && getFormatQuorter(value);
             }
-            if(fieldDef.numberType === NumberType.Multi){
-              return value && value + 'x'
+            if (fieldDef.numberType === NumberType.Multi) {
```
Comment (alissaVrk) [inline src/comps/grid/useGridColumns.tsx:120]: multi !== null there is no need for 2 ifs

File: src/comps/grid/useGridColumns.tsx [hunk 4]
```diff
@@ -103,14 +104,24 @@ export function useGridColumns(colsCategs: Array<ColumnCategory>, companiesData:
...
+              return value && formatNumberWithCommas(Number(value)) + "x";
+            }
+            if (fieldDef.numberType != null|| fieldDef.moneyType != null) {
```
Comment (alissaVrk) [inline src/comps/grid/useGridColumns.tsx:123]: there is a better way to check if it's number or money, the def has a type..

**Review by alissaVrk — CHANGES_REQUESTED** [review]

File: src/comps/grid/useGridColumns.tsx (hunk 4 above)
Comment (EyalMecilati) [reply src/comps/grid/useGridColumns.tsx:123]: That's what I'm using, no? fieldDef.numberType and fieldDef.moneyType

File: src/comps/grid/useGridColumns.tsx (hunk 3 above)
Comment (EyalMecilati) [reply src/comps/grid/useGridColumns.tsx:120]: what do you mean 2 ifs? for multi there is only one. and I can't get multi from fieldDef, I can only compare it to the fieldDef.numberType

File: src/comps/companyPage/columnsConfigAndUtils.ts [hunk 5]
```diff
@@ -102,7 +102,13 @@ export const columnsForLom = (product: ProductType) => {
 };
 
 export const formatNumberWithCommas = (num: number) => {
-  return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+  return num
+  ? num
+      .toFixed(1)
```
Comment (alissaVrk) [inline src/comps/companyPage/columnsConfigAndUtils.ts:107]: toFixed returns string as far as I know

**Review by alissaVrk — APPROVED** [review]

**Pull request merged**